wls-broadcast-service/src/test/java/de/muenchen/oss/wahllokalsystem/broadcastservice/security/BroadcastSecurityTest.java

```diff
@@ -45,10 +43,7 @@ class BroadcastTest {
 
         @Test
         void accessDenied_dummyRolle() {
-            SecurityContextHolder.getContext().setAuthentication(
-                    new UsernamePasswordAuthenticationToken(TestConstants.TESTUSER, TestConstants.TESTPASSWORD,
-                            AuthorityUtils.createAuthorityList("ROLE_DUMMY")));
-
+            SecurityUtils.runWith(AuthorityUtils.createAuthorityList("ROLE_DUMMY").toString());
```

Comment:
`ROLE_DUMMY` kanst du als Constante für den Testauslagern, ist aber nicht notwendig.

Wichtig ist dass du das `runWith` vereinfachen kannst:

```diff
-SecurityUtils.runWith(AuthorityUtils.createAuthorityList("ROLE_DUMMY").toString());
+SecurityUtils.runWith("ROLE_DUMMY");
```

Reply:
resolved in d75f875fee9fa7dd32ba227f2c93a671e9fa105b